### PR TITLE
feat(profiles): Add dataset configurations and processor for profile chunks

### PR DESCRIPTION
**Pull request by phacops**
Related:
- https://github.com/getsentry/sentry-kafka-schemas/pull/272
- https://github.com/getsentry/snuba/pull/5896
- https://github.com/getsentry/snuba/pull/5895

### CODE REVIEW

File: snuba/datasets/configuration/profiles/entities/chunks.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+version: v1
```
Comment (volokluev) [inline snuba/datasets/configuration/profiles/entities/chunks.yaml:1]: FYI: You no longer need this file! Storages can be queried directly now (but you can keep it if you want)

File: snuba/datasets/configuration/profiles/storages/chunks.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        chunk_id: null
+  - processor: TableRateLimit
+allocation_policies:
+  - name: ConcurrentRateLimitAllocationPolicy
+    args:
+      required_tenant_types:
+        - referrer
+        - project_id
+      default_config_overrides:
+        is_enforced: 0
```
Comment (volokluev) [inline snuba/datasets/configuration/profiles/storages/chunks.yaml:36]:
```suggestion
        is_enforced: 1
```

File: snuba/datasets/configuration/profiles/storages/chunks.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      default_config_overrides:
+        is_enforced: 1
+        throttled_thread_number: 1
+        org_limit_bytes_scanned: 100000
+  - name: ReferrerGuardRailPolicy
+    args:
+      required_tenant_types:
+        - referrer
+      default_config_overrides:
+        is_enforced: 0
```
Comment (volokluev) [inline snuba/datasets/configuration/profiles/storages/chunks.yaml:50]:
```suggestion
        is_enforced: 1
```

File: snuba/datasets/configuration/profiles/storages/chunks.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        is_active: 0
```
Comment (volokluev) [inline snuba/datasets/configuration/profiles/storages/chunks.yaml:51]:
```suggestion
```

Comment (volokluev) [inline snuba/datasets/configuration/profiles/storages/chunks.yaml:51]:
```suggestion
```

**Review by volokluev — APPROVED** [review]
Looks good, just turn on those allocation policies. You see them off by default to to a rollout a while ago. They're all on in prod now

**Review by Zylphrex — APPROVED** [review]

**Pull request merged**